pkg/execute/executor.go

```diff
@@ -256,10 +256,8 @@ func trimQuotes(clusterValue string) string {
 func runKubectlCommand(args []string, clusterName, defaultNamespace string, isAuthChannel bool) string {
 
 	// run commands in namespace specified under Config.Settings.DefaultNamespace field
-	if len(defaultNamespace) != 0 {
+	if !utils.Contains(args, "-n") && len(defaultNamespace) != 0 {
```

Comment:
Thank you @girishg4t for catching this. Namespace can also be passed with `--namespace` flag, let's handle that too.

Reply:
I forgot to add that, Thanks @PrasadG193